doc/index.md

```diff
@@ -38,10 +38,22 @@ and newer and can be installed with:
 ```{toctree}
 ---
 maxdepth: 2
+caption: Installation and License
 ---
 Installation
 ```
 
+## License
```

Comment:
Moved the license to the top as it was not very visible before and needing a scroll